src/Common/src/Interop/Windows/sspicli/SecuritySafeHandles.cs

```diff
@@ -838,41 +780,27 @@ public new IntPtr DangerousGetHandle()
             ref Interop.SspiCli.ContextFlags outFlags)
         {
 #if TRACE_VERBOSE
```

Comment:
I don't know.  I didn't want to remove any of these.  As I noted in the PR description, there is likely some tracing being done that's no longer necessary, and there's probably some missing tracing.  My goal here wasn't to address that, but rather to get tracing into a working form.  Subsequently someone can review all of what's actually being trace holistically.